src/screens/CreateTokenConfirm.js

```diff
@@ -163,7 +162,21 @@ class CreateTokenConfirm extends React.Component {
           onBackPress={() => this.props.navigation.goBack()}
           onCancel={() => this.props.navigation.dismiss()}
         />
-        {this.state.modal}
+
+        { this.state.modal && (
+          // eslint-disable-next-line react/jsx-indent
+          <SendTransactionFeedbackModal
+            text={this.state.modal.text}
+            sendTransaction={this.state.modal.sendTransaction}
+            promise={this.state.modal.promise}
+            successText={<TextFmt>{t`**${this.name}** created successfully`}</TextFmt>}
+            onTxSuccess={this.onTxSuccess}
+            onDismissSuccess={this.exitScreen}
+            onDismissError={() => this.setState({ modal: null })}
+            hide={this.props.isShowingPinScreen}
```

Comment:
What's the advantage of hiding it like this instead of just not instantiating the component here?
Is there one?

Something like changing the condition above:

```js
{ this.state.modal && !this.props.isShowingPinScreen && (
```